src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -1330,19 +1330,27 @@ protected override void Dispose(DisposeTypes type)
                 return;
             }
 
-            if (type == DisposeTypes.Explicit)
-            {
-                StopOverShootingShadowAnimation();
-                StopScroll();
+            StopOverShootingShadowAnimation();
+            StopScroll();
 
-                if (mPanGestureDetector != null)
-                {
-                    mPanGestureDetector.Detected -= OnPanGestureDetected;
-                    mPanGestureDetector.Dispose();
-                    mPanGestureDetector = null;
-                }
+            if (mPanGestureDetector != null)
+            {
+                mPanGestureDetector.Detected -= OnPanGestureDetected;
+                mPanGestureDetector.Dispose();
+                mPanGestureDetector = null;
+            }
 
+            if (propertyNotification != null)
+            {
+                propertyNotification.Notified -= OnPropertyChanged;
+                propertyNotification.RemovePropertyNotifications();
```

Comment:
`RemovePropertyNotifications` is member API of `Animatable.cs`.  Did you mean this code?
```suggestion
                RemovePropertyNotifications();
```